src/common/dav/dav_iface.c

```diff
@@ -313,9 +312,11 @@ dav_obj_close(dav_obj_t *hdl)
 	D_FREE(hdl->do_heap);
 
 	lw_tx_begin(hdl);
-	stats_delete(hdl, hdl->do_stats);
+	stats_persist(hdl, hdl->do_stats);
```

Comment:
Is that duplicated by "stats_persist(pop, pop->do_stats);" which is the first call inside lw_tx_end()?

Reply:
Removed this block of code as stats are a persisted every time wal commit is called.